xia2 no longer support Python 3.6
